conans/model/workspace.py

```diff
@@ -12,11 +12,13 @@
 
 
 class LocalPackage(object):
-    def __init__(self, base_folder, data, cache, ws_layout, ws_generators):
+    def __init__(self, base_folder, data, cache, ws_layout, ws_generators, ref):
+        if not data:
```

Comment:
Could we move here both checks to avoid the duplicated raise?
` if not data or not data.get("path"):` ?

Reply:
Not that easy, because I am not doing a ``data.get()``, but a ``data.pop()`` to extract the field and that cannot be done in a single line.

Reply:
Mhh I'm missing something probably, but if you first check if the data["get"] is there or not, the following check after the data.pop("get") is not necessary, right?

Reply:
Ok, right, done.